src/main/java/ru/skypro/flea/service/impl/ImageServiceImpl.java

```diff
@@ -0,0 +1,66 @@
+package ru.skypro.flea.service.impl;
+
+import lombok.Setter;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import org.springframework.web.multipart.MultipartFile;
+import ru.skypro.flea.service.ImageService;
+
+import javax.annotation.PostConstruct;
+import java.io.File;
+import java.io.IOException;
+import java.util.Set;
+
+@Slf4j
+@Service
+@Setter
+public class ImageServiceImpl implements ImageService {
+
+  private static final String IMAGES_PATH = "src/main/resources/images";
```

Comment:
there is no ability to test this class if you specify path like this
you'll need resources for tests that should be placed in resources of test catalogue
so the path will not be the same

i suggest you to set path that way:
```java
private String imagesPath;

public ImageServiceImpl() {
  imagesPath = "src/main/resources/images";
}

public ImageServiceImpl(String imagesPath) {
  this.imagesPath = imagesPath;
}
```
so, first constructor will be used in runtime
and you can use second constructor for tests:
```java
class ImageServiceTest {

    private final ImageServiceImpl out = new ImageServiceImpl("/path/to/test/resources/that/need/to/be/specified");

    @Test
    public void test() {
        // ...
    }

}
```

i guess it is not the best way to do this, but it works